analyzers/src/SonarAnalyzer.Common/Helpers/ComparisonKinds.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+
+namespace SonarAnalyzer.Helpers
+{
+    public static class ComparisonKinds
+    {
+        public static IReadOnlyCollection<ComparisonKind> All => new[]
```

Comment:
I don't see where this will be used in S3981. I'd remove this file. Same collection can be generated with this and stored to a static field in the rule itself.
```
System.Enum.GetValues(typeof(ComparisonKind)).Cast<ComparisonKind>().Where(x => x != ComparisonKind.None)
```

Reply:
Updated.